File: Basic/JS/1011.js
/**
 * @author: kurumi
 * @date: 2025-10-25
 * @description: 1011 A+B 和 C
 */
const rl = require("readline").createInterface({input: process.stdin});
const iter = rl[Symbol.asyncIterator]();
const readline = async () => (await iter.next()).value;

void (async function() {
  const n = parseInt(await readline());
  for (let i = 1; i <= n; i++) {
    const nums = (await readline()).split(" ").map(BigInt);
    console.log(`Case #${i}: ${nums[0] + nums[1] > nums[2]}`);
  }
})();

File: Basic/Python/1011.py
'''
@Author: kurumi
@Date: 2025-10-25
@Description: 1011 A+B 和 C
'''
def main():
    n: int = int(input())
    for i in range(1, n + 1):
        a, b, c = map(int, input().split())
        print(f'Case #{i}: {"true" if a + b > c else "false"}')

if __name__ == '__main__':
    main()